communication/README.md

```diff
@@ -36,6 +36,16 @@ devoted to the Kubernetes project. Please check the guidelines and any relevant
 chat/conversation history before posting. Spam and sales pitches are not tolerated
 on these platforms.
 
+### Appropriate Content for Community Resources
+
+All communications properties are under the [Kubernetes code of conduct]. 
+Additionally, these resources are for the contributors and users of Kubernetes, commercial usage of these properties is heavily moderated.
+
+Examples of inappropriate content:
+- Posting content of a commercial nature on Slack or the Forums
```

Comment:
Why is Forums capitalized?

Reply:
Should we maybe change the phrasing to be more generic? "Slack or other community forums"?

Reply:
re: commercial - what about the managed service vendors slack channels, etc?

Reply:
I've made it more generic, and added some examples of vendor slack channels in my latest push.